marketstack/package.json

```diff
@@ -1,15 +1,46 @@
 {
   "name": "@chainlink/marketstack-adapter",
   "version": "0.0.1",
+  "description": "Chainlink marketstack adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "marketstack"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
   "license": "MIT",
-  "main": "index.js",
   "scripts": {
-    "server": "node -e 'require(\"./index.js\").server()'",
+    "prepublishOnly": "yarn build && yarn test:unit",
+    "setup": "yarn build",
+    "build": "tsc -b",
     "lint": "eslint --ignore-path ../.eslintignore . --ext .js,.jsx,.ts,.tsx",
     "lint:fix": "eslint --ignore-path ../.eslintignore . --ext .js,.jsx,.ts,.tsx --fix",
-    "test": "yarn _mocha --timeout 0",
-    "test:unit": "yarn _mocha --grep @integration --invert --timeout 0",
-    "test:integration": "yarn _mocha --grep @integration --timeout 0"
+    "test": "mocha --exit --timeout 0 -r ts-node/register 'test/**/*.test.ts'",
+    "test:unit": "mocha --exit --grep @integration --invert -r ts-node/register 'test/**/*.test.ts'",
+    "test:integration": "mocha --exit --timeout 0 --grep @integration -r ts-node/register 'test/**/*.test.ts'",
+    "server": "node -e 'require(\"./index.js\").server()'",
```

Comment:
```suggestion
```
Take these out of every TS migrated EA as `./index.js` is now gone.

Reply:
Taking it out of the `example` as well.